google-scripts/process-form-submission/getProperties.ts

```diff
@@ -0,0 +1,35 @@
+export function getProperties() {
+  const REFERRALS_SHEET_NAME =
```

Comment:
Perhaps we could condense the below slightly?

```ts
const { getProperty } = PropertiesService.getScriptProperties();

const REFERRALS_SHEET_NAME = getProperty("MASH_SHEET_NAME");

// ... and so on
```

Reply:
Done in 2eb2227